lib/matplotlib/axes/_base.py

```diff
@@ -2617,6 +2617,38 @@ def use_sticky_edges(self, b):
         self._use_sticky_edges = bool(b)
         # No effect until next autoscaling, which will mark the Axes as stale.
 
+    def get_xmargin(self):
+        """
+        Retrieve autoscaling margin of the x-axis.
+
+        .. versionadded:: 3.7
```

Comment:
This would be in 3.9, as 3.7 is already out:
```suggestion
        .. versionadded:: 3.9
```